ospo-mindmap/Content/ospomindmap.md

```diff
@@ -109,24 +189,87 @@ incubation and post launch
   - Vet, organize and operate OS projects
   - Train and coach OS Leaders
 
-### OS Leadership
+#### Optimize Open Outbound Source Contributions
+
+
+### 🤝 Collaborate with Open Source Organizations
+
+##### 📖 Open Standards
+
+- OpenChain
+- SPDX
+
+##### 🏠 Foundations
+
+- Apache Foundation
+- Free Software Foundation Europe
+  - Legal Network
+- Eclipse Foundation
```

Comment:
Under Eclipse Foundation one can add:

- OSPO Alliance (https://ospo.zone/)

Reply:
IMHO, I would like to see more content and resources/tools being produced/announced by this early-stage community before adding this as a project. Right now everything looks opaque but the OW2's Good Governance handbook (which AFAIK is already added in the OW2 section).  (cc @tsteenbe for input here)